.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -164,8 +164,8 @@ const getPipeline = (filename: string, removeSteps = true) => {
     if (
       (await doAnyChangesMatch([
         /^x-pack\/platform\/packages\/shared\/ai-infra/,
-        /^x-pack\/platform\/plugins\/shared\/ai_infra/,
-        /^x-pack\/platform\/plugins\/shared\/inference/,
+        /^x-pack\/platform\/platform\/plugins\/shared\/shared\/ai_infra/,
```

Comment:
These paths don't look right, the "platform" part seems duplicated. (More occurrences in the file)

Reply:
Thank you! adjusted manually

cc @gsoldevila